common/lib/capa/capa/tests/test_hint_functionality.py

```diff
@@ -65,21 +65,21 @@ def test_tracking_log(self):
 
     @data(
         {'problem_id': u'1_2_1', u'choice': u'GermanyΩ',
-         'expected_string': u'<div class="feedback-hint-incorrect"><div class="hint-label">Incorrect: </div><div class="hint-text">I do not think so.Ω</div></div>'},
+         'expected_string': u'<div class="feedback-hint-incorrect"><div class="hint-label">Incorrect: </div><div class="hint-text">I do not think so.&#937;</div></div>'},
```

Comment:
The function etree.tostring that was used to parse the xpath nodes encodes special characters as the html number variant. 
So the greek symbol Ω gets encoded as the html number \&#937;  which is still rendered in browser as &#937; .
(I had to escape the & for the latter not to be shown as a symbol)
Here are twoimages from studio and lms to show it still renders as it should:
###### studio view:
![editor1](https://cloud.githubusercontent.com/assets/5675577/9521944/10701442-4cd2-11e5-87df-3a365aad4abe.png)
###### lms view:
![lms](https://cloud.githubusercontent.com/assets/5675577/9521945/1555f472-4cd2-11e5-887a-42809a31afd2.png)